velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -1531,6 +1533,149 @@ TpchPlan TpchQueryBuilder::getQ19Plan() const {
   return context;
 }
 
+TpchPlan TpchQueryBuilder::getQ21Plan() const {
+  std::vector<std::string> supplierColumns = {
+      "s_nationkey", "s_name", "s_suppkey"};
+  std::vector<std::string> lineitemColumnsWithDate = {
+      "l_suppkey", "l_commitdate", "l_orderkey", "l_receiptdate"};
+  std::vector<std::string> lineitemColumns = {"l_suppkey", "l_orderkey"};
+  std::vector<std::string> ordersColumns = {"o_orderkey", "o_orderstatus"};
+  std::vector<std::string> nationColumns = {"n_nationkey", "n_name"};
+
+  auto supplierSelectedRowType = getRowType(kSupplier, supplierColumns);
+  const auto& supplierFileColumns = getFileColumnNames(kSupplier);
+  auto lineitemOneSelectedRowType =
```

Comment:
Thanks for the suggestion, I had named it this way because of a review comment by Masha on another PR where I was asked not to name variables as var1, var2 etc,. I will use the numbers as suggested.